asab/log.py

```diff
@@ -470,6 +470,9 @@ def _connect(self, loop):
 		self._loop.add_writer(self._socket, self._on_write)
 		self._loop.add_reader(self._socket, self._on_read)
 
+	def _reconnect(self):
+		self._loop.call_later(1, self._connect, self._loop)
```

Comment:
This is the most naive implementation of reconnect using a reasonable delay.
If we agree this is a good approach, I can implement prolonging of the delay with number of retries and max retries.